dss/operations/lambda_params.py

```diff
@@ -35,21 +35,27 @@ def get_ssm_variable_prefix() -> str:
     """
     store_name = os.environ["DSS_PARAMETER_STORE"]
     stage_name = os.environ["DSS_DEPLOYMENT_STAGE"]
-    store_prefix = f"{store_name}/{stage_name}"
+    store_prefix = f"/{store_name}/{stage_name}"
     return store_prefix
 
 
 def fix_ssm_variable_prefix(param_name: str) -> str:
     """Add (if necessary) the variable store and stage prefix to the front of the name of an SSM store parameter"""
     prefix = get_ssm_variable_prefix()
 
-    # Strip leading/trailing slash
+    # Strip leading slash
     if param_name.startswith("/"):
         param_name = param_name[1:]
+
+    # Strip trailing slash
     if param_name.endswith("/"):
         param_name = param_name[:-1]
 
-    if not (param_name.startswith(prefix) or param_name.startswith("/" + prefix)):
+    if not (
+            param_name.startswith(prefix)
+            or param_name.startswith("/" + prefix)
+            or param_name.startswith(prefix[1:])
+    ):
         param_name = f"{prefix}/{param_name}"
```

Comment:
Is this equivalent?:
```
return param_name.replace(prefix, "", 1).lstrip("/")
```

Note that `lstrip("/")` will strip multiple slashes off the left.

Reply:
Excellent, thank you Brian

Gonna start calling you Mr. brevity is the soul of wit